compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__leak_multiple_private_types.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: compiler-core/src/type_/tests/errors.rs
+expression: "\n        type Private {\n            Private\n        }\n\n        pub fn ret_private() -> Private {\n            Private\n        }\n\n        pub fn ret_private2() -> Private {\n            Private\n        }\n\n        pub fn main() {\n            ret_private()\n        }\n        "
+---
+error: Private type used in public interface
+  ┌─ /src/one/two.gleam:6:9
+  │
+6 │         pub fn ret_private() -> Private {
+  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following type is private, but is being used by this public export.
+
+    Private
+
+Private types can only be used within the module that defines them.
+error: Private type used in public interface
+   ┌─ /src/one/two.gleam:10:9
+   │
+10 │         pub fn ret_private2() -> Private {
+   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following type is private, but is being used by this public export.
+
+    Private
+
+Private types can only be used within the module that defines them.
+error: Private type used in public interface
```

Comment:
Let's put an empty line between errors to make it easier to see where they start and end